arquillian/arquillian-tomee-tests/arquillian-tomee-webprofile-tests/src/test/resources/hibernate-pom.xml

```diff
@@ -23,11 +23,15 @@
   <version>1.0.0</version>
   <artifactId>hibernate-deps</artifactId>
 
+  <properties>
+    <version.hibernate.orm>6.1.2.Final</version.hibernate.orm>
```

Comment:
We have `6.1.3.Final` available now: https://mvnrepository.com/artifact/org.hibernate.orm/hibernate-core/6.1.3.Final
Perhaps we can update directly to `6.1.3.Final` ?

Reply:
you are right, i'll make a change !